server/models/Card.js

```diff
@@ -1,51 +1,42 @@
 const mongoose = require("mongoose");
 
-const cardSchema = new mongoose.Schema({
-  _id: {
-    type: mongoose.Types.ObjectId,
-    required: true,
-  },
-  name: {
-    type: Array,
-    required: false,
-  },
-  tag: {
-    type: String,
-    required: true,
-  },
-  description: {
-    type: String,
-    required: false,
-  },
-  deadline: {
-    type: Date,
-    required: false,
-  },
-  comment: {
-    type: String,
-    required: false,
-  },
-  attachment: {
-    type: String,
-    required: false,
-  },
-  checklist: {
-    type: Array,
-    required: false,
-  },
-  columnId: {
-    type: mongoose.Types.ObjectId,
-    required: true,
-  },
-  index: {
-    type: Number,
-    required: true,
-  },
-  createdAt: {
-    type: Date,
-    default: Date.now,
-  },
-});
+const cardSchema = new mongoose.Schema(
+  {
+    name: {
+      type: Array,
+      required: false,
+    },
+    tag: {
+      type: String,
+      required: true,
+    },
+    description: {
+      type: String,
+      required: false,
+    },
+    deadline: {
+      type: Date,
+      required: false,
+    },
+    comment: {
+      type: String,
+      required: false,
+    },
+    attachment: {
+      type: String,
+      required: false,
+    },
+    checklist: {
+      type: Array,
+      required: false,
+    },
+    columnId: {
+      type: mongoose.Types.ObjectId,
+      required: true,
+    },
```

Comment:
Adding a `ref: "column"` will ensure only a Column ID can be populated for future querying with Cards.